10up/simple-podcasting
# Modernize the build tools

### Description of the Change
This PR updates the plugin build tools to use `@wordpress/scripts` which results in a simple and effective build configuration.



### Alternate Designs

I considered using `10up-toolkits`, but I decided to use `wp-scripts` because the latter is a more community-used tool.



### Benefits

Make our build works with recent `node` versions and other tools/packages while reducing the complexity of configuration.



### Possible Drawbacks
n/a


### Verification Process
1. Remove `node_modules`.
2. Check out this PR.
3. Run `npm i`.
4. Run `npm run build`, see assets build correctly into `dist` folder.
5. Add podcast/category, see plugin's functions work as before, see no error in the console/ debug log.



### Checklist:






### Applicable Issues
Fixes #113


### Changelog Entry
Updated: use `@wordpress/scripts` as the build tool.
